left_panda:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 500  # in Hz


  joint_position_controller:
    type: position_controllers/JointGroupPositionController
    joints:
      - left_panda_joint1
      - left_panda_joint2
      - left_panda_joint3
      - left_panda_joint4
      - left_panda_joint5
      - left_panda_joint6
      - left_panda_joint7


right_panda:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 500  # in Hz


  joint_position_controller:
    type: position_controllers/JointGroupPositionController
    joints:
      - right_panda_joint1
      - right_panda_joint2
      - right_panda_joint3
      - right_panda_joint4
      - right_panda_joint5
      - right_panda_joint6
      - right_panda_joint7

  # gazebo_ros_control:
  #   pid_gains:
  #     panda_joint1: {p: 50, i: 0, d: 20}
  #     panda_joint2: {p: 10000, i: 0, d: 500}
  #     panda_joint3: {p: 100, i: 0, d: 10}
  #     panda_joint4: {p: 1000, i: 0, d: 50}
  #     panda_joint5: {p: 100, i: 0, d: 10}
  #     panda_joint6: {p: 100, i: 0, d: 10}
  #     panda_joint7: {p: 10, i: 0.5, d: 0.1}
  #     panda_finger_joint1: {p: 100, i: 0, d: 50}
  #     panda_finger_joint2: {p: 100, i: 0, d: 50}